UPDATING.md

```diff
@@ -52,6 +52,39 @@ assists users migrating to a new version.
 
 ## Master
 
+### Changes to output argument in commands
+
+Instead of using [tabulate](https://pypi.org/project/tabulate/) to render commands output
+we use [rich](https://github.com/willmcgugan/rich). Due to this change the `--output` argument
+will no longer accept formats of tabulate tables. Instead it accepts:
+
+- `table` - will render the output in predefined table
+- `json` - will render the output as a json
+- `yaml` - will render the output as yaml
+
+By doing this we increased consistency and gave users possibility to manipulate the
+output programmatically (when using json or yaml).
+
+Affected commands:
+
+- `airflow dags list`
+- `airflow dags report`
+- `airflow dags list-runs`
+- `airflow dags list-jobs`
+- `airflow connections list`
+- `airflow connections get`
+- `airflow pools list`
+- `airflow pools get`
+- `airflow pools set`
+- `airflow pools delete`
+- `airflow pools export`
+- `airflow role list`
+- `airflow providers list`
+- `airflow providers get`
+- `airflow tasks states-for-dag-run`
+- `airflow users list`
+- `airflow variables list`
```

Comment:
`airflow pools import` and `airflow providers hooks` should be in this list as well?

Reply:
Done in caf81ff

Reply:
@turbaszek after your latest change (some commands don't print table anymore), maybe we need to further update this doc as well.